cmd/config-current.go

```diff
@@ -109,8 +109,12 @@ func (s *serverConfig) SetStorageClass(standardClass, rrsClass storageClass) {
 	defer s.Unlock()
 
 	// Set the values
-	s.StorageClass.Standard = standardClass.Scheme + strconv.Itoa(standardClass.Parity)
-	s.StorageClass.RRS = rrsClass.Scheme + strconv.Itoa(rrsClass.Parity)
+	if standardClass.Parity != 0 {
+		s.StorageClass.Standard = standardClass.Scheme + ":" + strconv.Itoa(standardClass.Parity)
```

Comment:
We should avoid this by adding a `storageClass.String()` method to get the `scheme:N`  directly this type of problem can be avoided. Then you can simply add `s.StorageClass.Standard  = standardClass.String()`

Reply:
are you going to do these changes? @nitisht

Reply:
yes @harshavardhana , @krisis had some suggestions - I am working on that, will update the PR